src/Logic/OCR/OcrFixEngine.cs

```diff
@@ -347,12 +347,8 @@ public string SpellCheckDictionaryName
 
         public string FixOcrErrors(string text, int index, string lastLine, bool logSuggestions, AutoGuessLevel autoGuess)
         {
-            while (text.Contains(Environment.NewLine + " "))
-                text = text.Replace(Environment.NewLine + " ", Environment.NewLine);
-            while (text.Contains(" " + Environment.NewLine))
-                text = text.Replace(" " + Environment.NewLine, Environment.NewLine);
-            while (text.Contains(Environment.NewLine + Environment.NewLine))
-                text = text.Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);
+            text = text.FixResursiveLineBreaks();
+            text = text.FixExtraSpaces();

```

Comment:
Note: FixExtraSpaces will remove both **recursive-whitespace** and white-spaces after '\r' or '\n' so...